devtools/gradle/src/test/resources/gradle-project/build.gradle

```diff
@@ -5,9 +5,7 @@ plugins {
 }
 
 repositories {
-    maven {
-        url uri(System.getenv('MAVEN_LOCAL_REPO'))
-    }
+    mavenLocal()
```

Comment:
Are we sure this won't be an issue when I'm building things with a specific Maven repo (i.e. when I release)?

Reply:
Yes, because `mavenLocal()` reads the `maven.repo.local` that is set in https://github.com/quarkusio/quarkus/pull/5761/files#diff-32bff5c9d7d29fc887c0eb01a4f45724R67